cc/README.md

```diff
@@ -9,115 +10,188 @@ build system. We have experienced build issues with the latest version of Bazel,
+Minigo++ depends on the TensorFlow C++ libraries, but we have not yet set up
 Bazel WORKSPACE and BUILD rules to automatically download and configure
-Tensorflow so (for now at least) you must perform a manual step to build the
+TensorFlow so (for now at least) you must perform a manual step to build the
 library.  This depends on `zip`, so be sure that package is installed first:
 
 ```shell
 sudo apt-get install zip
 ./cc/configure_tensorflow.sh
 ```
 
-If you want to compile for CPU and not GPU, then change `TF_NEED_CUDA` to 0 in
-`configure_tensorflow.sh`
+If you want to compile for CPU and not GPU, then execute the following instead:
+
+```shell
+sudo apt-get install zip
+TF_NEED_CUDA=0 ./cc/configure_tensorflow.sh
+```
 
 This will automatically perform the first steps of
-[Installing Tensorflow from Sources](https://www.tensorflow.org/install/install_sources)
-but instead of installing the Tensorflow package, it extracts the generated C++
+[Installing TensorFlow from Sources](https://www.tensorflow.org/install/install_sources)
+but instead of installing the TensorFlow package, it extracts the generated C++
 headers into the `cc/tensorflow` subdirectory of the repo. The script then
-builds the required Tensorflow shared libraries and copies them to the same
+builds the required TensorFlow shared libraries and copies them to the same
 directory. The tensorflow cc\_library build target in `cc/BUILD` pulls these
-header and library files together into a format the Bazel understands how to link
-against.
+header and library files together into a format the Bazel understands how to
+link against.
 
-## Building
-
-The C++ Minigo implementation requires that the board size be defined at compile
-time, using the `MINIGO_BOARD_SIZE` preprocessor define. This allows us to
-significantly reduce the number of heap allocations performed. The build scripts
-are configured to compile with `MINIGO_BOARD_SIZE=19` by default. To compile a
-version that works with a 9x9 board, invoke Bazel with `--define=board_size=9`.
+Although Minigo uses TensorFlow as its inference engine by default, other
+engines can be used (e.g. Cloud TPU, TensorRT, TensorFlow Lite). See the
+Inferences engines section below for more details.
 
-## Running the unit tests
+## Getting a model
 
-Minigo's C++ unit tests operate on both 9x9 and 19x19, and some tests are only
-enabled for a particular board size. Consequently, you must run the tests
-multiple times:
+The TensorFlow models can found on our public Google Cloud Storage bucket. A
+good model to start with is 000990-cormorant from the v15 run. The C++ engine
+requires the frozen model in a `GraphDef` proto format, which is the `.pb` file.
+You can copy it locally as follows:
 
 ```shell
-bazel test --define=board_size=9 cc/...  &&  bazel test cc/...
+mkdir -p saved_models
+gsutil cp gs://tensor-go-minigo-v15-19/models/000990-cormorant.pb saved_models/
```

Comment:
Needs a public bucket link

Reply:
I shouldn't really just copy stuff out of my bash history :)